gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,650 @@
+
+
+[[troubleshooting]]
+== Troubleshooting
+//iterative troubleshooting info to add as needed:
+
+If you encounter an error when running a command, check for troubleshooting information: 
+
+<<docker-troubleshooting>> | <<helm-troubleshooting>>
+
+[[docker-troubleshooting]]
+=== Docker Command Issues
+
+Troubleshoot errors returned by a `docker` command:
+
+[[daemon-bad-response]]
+* If Docker is not started, a `docker` command returns an error message similar to these:
++
+----
+Error response from daemon: Bad response from Docker engine
+----
++
+If you get this error, you need to start Docker.
+
+
+[[bad-request]]
+* The following error can occur when you use `docker run` to register your gateway:
++
+----
+[flexctl][error] reg facade call returned error response:
+HTTP/1.1 400 Bad Request
+----
++
+The reason for this error is available within the error message. Common causes include: 
+
+[[name-already-exists]]
+** `an active target with the same name already exists in this organization and environment`. 
++
+To avoid this issue, provide a _unique name_ for your gateway when running the <<reg-command, registration command>>. Note that you _cannot reuse the same name_ even if a previous registration with that name was successful. When this error occurs, the command generates an _empty_ `registration.yaml`, which prevents re-registration _and_ produces an error when you try to use the file when deploying the gateway to your cluster. 
+
+[[no-token]]
+** `no valid registration token was found`
++
+To avoid this issue, generate a new token for the command by refreshing the *Add a Flex Gateway* page for Kubernetes and rerun the <<reg-command, registration command>>.
+
+
+[[helm-troubleshooting]]
+=== Helm Command Issues
+
+Troubleshoot errors returned by a `helm` command:
+
+[[yaml-empty]]
+* If the `registration.yaml` file is invalid when you pass it to the Helm command during deployment of Flex Gateway, the following error results:
++
+----
+Error: UPGRADE FAILED: execution error at (flex-gateway/templates/deployment.yaml:4:10):
+registerSecretName, registration.content or registration.secretName is required!
+----
++
+When this error occurs, check the `registration.yaml`. Make sure the file is not empty. If the file is empty, re-register your gateway with a new, unique name to generate a new `registration.yaml` (see <<register-flex>>) and rerun the procedures that follow the registration step. 
+
+[[cluster_unreachable]]
+* If your Kubernetes cluster is not running when you attempt to deploy Flex Gateway using `helm`, the following error can occur:
++
+----
+Error: Kubernetes cluster unreachable: 
+the server could not find the requested resource
+----
++
+Start the service that runs your cluster. 
++
+You can check that your Kubernetes cluster host is running your cluster, for example:
++
+[source,kubectl]
+----
+kubectl cluster-info
+----
++
+If the cluster is running, the command returns a result similar to this one:
```

Comment:
```suggestion
If the cluster is running, the command returns a result similar to:
```

Reply:
(I avoid "one" - I think I suggested "the following" earlier, but I think it's obvious from the colon which one you mean.)

Reply:
Is there guidance on this?